generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management.tsx

```diff
@@ -33,14 +35,91 @@ export interface IUserManagementProps {
   match: any;
 }
 
-export class UserManagement extends React.Component<IUserManagementProps, undefined> {
+export interface IUserManagementState {
+  itemsPerPage: number;
```

Comment:
these will be common for all entities, hence please make it a common interface and extend from that here

Reply:
I wanted validate with you the mechanism (pagination) only on this screen to begin

Reply:
yes we should get it right here first then apply the same for entity :+1: